coalib/testing/LocalBearTestHelper.py

```diff
@@ -25,6 +25,17 @@ def execute_bear(bear, *args, **kwargs):
                 console_output.append('The program yielded '
                                       'the following output:\n')
```

Comment:
I guess we have to move this text into `new_process_output`, so the text order makes more sense.
For that we should maybe Replace `Arguments` with `Program arguments` because that's the first the developer reads